saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/property/PairwiseConjunction.scala

```diff
@@ -15,8 +15,8 @@ object PairwiseConjunction {
   def concat[T](a: Property[T], b: List[Property[T]], y: T): List[String] =
```

Comment:
Pending. Will add in a separate PR. Don't understand what this currently does. Will add a test as well.